modules/cardano-staging.nix

```diff
@@ -8,5 +8,7 @@ params:
     ./papertrail.nix
   ];
   
+  services.monitoring-exporters.graylogHost = "${config.deployment.arguments.globals.monitoringNV.name}-ip:5044";
```

Comment:
If we move this assignment one abstraction level higher, we'll have less files to need to include it in.  The iohk/common/Constants.hs file shows monitoring configured for the following environments at present: devops, testnet, staging and production.  If we then move this assignment out of this file and instead to each of those respective environment files (modules/{devops,testnet,staging,production}.nix), I think we should be covered.  Would that work for you?